vnext/Microsoft.ReactNative.Cxx/ReactContext.h

```diff
@@ -8,6 +8,7 @@
 #include <string_view>
 #include "JSValueWriter.h"
 #include "winrt/Microsoft.ReactNative.h"
+#include <../include/CppWinRTIncludes.h>
```

Comment:
We need to find a way to make this file to be part of the Microsoft.ReactNative.Cxx shared project. We will continue to distribute this code as sources after we switch the rest to DLL distribution. Thus this code cannot depend on any 'internal' implementation files.

Reply:
this file existed before this PR :) please file an issue to track figuring out where this file should be.